### DISCUSSION TITLE
GitHub Education

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#35054] [New to GitHub] [answered]
Hey everyone,

Can someone please let me know what are issues section in GitHub is?

**Comment by DhanushNehru** [comment, marked as answer]
Issues let you track your work on GitHub, where development happens. Basically it tells you what are the issues found on the current repository

When you mention an issue in another issue or pull request, the issue's timeline reflects the cross-reference so that you can keep track of related work.

To indicate that work is in progress, you can link an issue to a pull request.
